src/components/attribute/PullRequest.js

```diff
@@ -60,7 +60,7 @@ export const PullRequest = ({ record, pr }) => {
       )}
       {!loading && (!authed || !fetchedPr) && (
         <span className="pr-status">
-          <aha-button onClick={fetchData} size="small" type="attribute">
+          <aha-button onClick={fetchData} size="small" kind="attribute">
```

Comment:
@nullset Did you mean secondary here?

```suggestion
          <aha-button onClick={fetchData} size="small" kind="secondary">
```

Reply:
Also, the icon is barely visible:

![Screen Shot 2022-08-16 at 9 56 11 AM](https://user-images.githubusercontent.com/125077/184924693-b539ee07-026d-4b5e-a885-f7659425e35d.png)

Reply:
Changing it to secondary seems to fix the icon colour issue.